package.json

```
@@ -1,10 +1,14 @@
 {
-  "version": "3.0.0",
+  "version": "3.1.0",
   "name": "@aeternity/aepp-components",
   "main": "dist/aepp.components.js",
   "description": "Reusable Vue Components for aeternity aepp projects",
   "homepage": "https://github.com/aeternity/aepp-components#readme",
-  "author": "Max Dörfler <max@apeunit.com> (https://github.com/muxe)",
+  "contributors": [
+    "Max Dörfler <max@apeunit.com> (https://github.com/muxe)",
+    "Denis Davidyuk <denis@apeunit.com> (https://github.com/davidyuk",
```

I don't have an email at apeunit.com, so:
```suggestion
    "Denis Davidyuk <denis_davidyuk@hotmail.com> (https://github.com/davidyuk)",
```

---

fixed